app/bundles/CoreBundle/Templating/Twig/Extension/EmojiExtension.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\CoreBundle\Templating\Twig\Extension;
+
+use Mautic\CoreBundle\Helper\EmojiHelper;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class EmojiExtension extends AbstractExtension
+{
+    protected EmojiHelper $emojiHelper;
+
+    public function __construct(EmojiHelper $emojiHelper)
+    {
+        $this->emojiHelper = $emojiHelper;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions()
+    {
+        return [
+            new TwigFunction('toHtml', [$this, 'toHtml'], ['is_safe' => ['all']]),
+            new TwigFunction('instanceof', [$this, 'isInstanceof']),
```

Comment:
This function is defined above and as I mentioned, it's already present in the 5.x branch, so can be removed from here too.